src/FileSystemWatcher.cpp

```diff
@@ -0,0 +1,38 @@
+#include "../includes/FileSystemWatcher.h"
+
+using namespace NodeSentinalFileWatcher;
+
+FileSystemWatcher::FileSystemWatcher(const std::string &path, std::chrono::milliseconds sleepDuration)
+    : _nativeInterface(new NativeInterface(path))
+    , _sleepDuration(sleepDuration)
+{
+  _runner = std::thread([this] {
```

Comment:
please put that in an extra member function for readability.

Reply:
i think a std::bind is a backward step
see cppcon talk from 2015 (https://www.youtube.com/watch?v=zt7ThwVfap0&t=32m19s)